src/brasil/gov/tiles/static/tiles.css

```diff
@@ -1716,6 +1716,18 @@ div[data-tile*="collective.cover.embed"] .samba-player embed {
   -moz-box-sizing: border-box;
 }
 
+/*XXX: Precisamos esconder o tile de nitf de collective.nitf, pois iremos usar*/
```

Comment:
eu acho que seria melhor fazer override da declaração do tile para usar o mesmo nome, com a nova classe e template; precisariam um upgrade step para procurar em todas as capas e trocar as referencias to tile atual para o nome `collective.nitf`.

dessa forma se no futuro as mudanças forem desnecessárias poderiam continuar usando o tile padrão.